CHANGELOG.md

```diff
@@ -1,6 +1,34 @@
 ## current master
 
-* update ohsome parent to version 2.3 (for building javadoc with JDK 11)
+
+## 1.1
+
+### Prominent Changes 
+
+* making filter2 parameter mandatory for /ratio requests and changing filter to be allowed to be null or empty (2dda1f8)
```

Comment:
ah, forgot this one: I don't think a user would know what `null` means in this context. maybe it would be better to use the word `omitted`, as in: _“…chaging filter to be allowed to be empty or omitted.”_